application/controllers/QuestionGroupsAdministrationController.php

```diff
@@ -367,8 +367,8 @@ public function actionListquestiongroups($surveyid)
         $baselang = $survey->language;
         $model    = new QuestionGroup('search');
 
-        if (isset($_GET['QuestionGroup'])) {
-            $model->attributes = $_GET['QuestionGroup'];
+        if (isset($_GET['QuestionGroup']['group_name'])) {
```

Comment:
Can we move to 
```
$QuestionGroup = App()->getRequest()->get('QuestionGroup'); 
if(!empty($QuestionGroup['group_name'])) {
```

Reply:
why?

Reply:
https://stackoverflow.com/questions/3498207/is-using-superglobals-directly-good-or-bad-in-php#3498463
https://phpmd.org/rules/controversial.html

Since we have a framework : use it …

Reply:
Maybe not super critical here, but yes, in general, using the request object is much, much better for testability than accessing super-globals.

Reply:
See «maybe (unneded)»

Better but uneeded

Reply:
OK. Great. Yes, code was like that. We didn't refactored

Reply:
Yes true :) isset came from previous code.